backend/services/src/main/java/com/crimealert/controllers/CommentController.java

```diff
@@ -68,19 +75,25 @@ public Response updateComment(
 	}
 	
 	@DELETE
-    @Consumes(MediaType.APPLICATION_JSON)
     @Produces(MediaType.APPLICATION_JSON)
+	@Path("/{commentId}")
 	public Response deleteComment(
-			@PathParam("commentId")String commentId
+			@NotNull@PathParam("commentId")String commentId
 	) {
     	try {
     	
+    		if (!commentId.isEmpty()){
```

Comment:
Looks like there is also a @NotEmpty constraint available - https://www.baeldung.com/java-bean-validation-not-null-empty-blank